.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: node_js
 node_js:
-  - "4.5"
+  - "12"
```

Comment:
Should it LTS then so that the API can be LTS?

Reply:
Isn't lts what's messing everything up?

Reply:
I changed the portal api back to LTS i just haven't pushed it yet

Reply:
Cool, honestly, whatever works.